corehq/apps/app_manager/templates/app_manager/partials/multimedia_sizes.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+<b>{% trans 'Version' %} {{ version }}</b>
+<div data-bind="if: load_state() === 'loading'">
+  <i class="fa fa-spin fa-spinner"></i>
+  {% trans "Loading Multimedia Sizes..." %}
+</div>
+<div data-bind="if: load_state() === 'loaded'">
+  <span data-bind="visible: !sizes().length" style="display:none">
```

Comment:
If you add the class `ko-template` to the `multimedia-sizes-container`, the whole thing will stay hidden until the knockout bindings are applied, and then you could remove the inline styles.

Reply:
nice!

Reply:
https://github.com/dimagi/commcare-hq/pull/25864/commits/eefa3803b051b2cf711a4216499ad14e31582843